### PR TITLE
Add a file with a list of expectations for different roles

### PR DESCRIPTION
**Pull request by locks**
@hergaiety I realized we didn't have a good place to point people to regarding expectations so I thought we could start up a document.

### CODE REVIEW

File: resources/expectations.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Expectations
+
+This document outlines what is expected from the different roles involved in running the workshop, in terms of time and tasks.
+
+## Hosts
+
+1. Arranging mentors (5h)
+
+You are responsible for arranging enough mentors to help contributors during the workshop.
+While a ratio of 1 mentor to 1 contributor is unrealistic, we recommend to keep it under 1 mentor to 5 contributors.
```
Comment (hergaiety) [inline resources/expectations.md]:
```suggestion
While a ratio of 1 mentor to 1 contributor is unrealistic, we recommend to strive for at least 1 mentor to 5 every contributors.
```
This read a little strangely, so here's a suggested alternative.

File: resources/expectations.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Preparing the introduction material (1h)
+
+Once you have hosts and mentors defined you should update the introductory slides to include that information.
+
+1. Running the workshop (5h)
+
+On the day of the actual workshop you should outline the workshop format and introduce the different mentors.
+Then try to keep an eye on people having a hard time participating or finding work to do and try to help out.
+We recomend a 3h format, and that you leave yourself some prep and wind down time.
```
Comment (hergaiety) [inline resources/expectations.md]:
```suggestion
We recommend a 3h format as well as leaving yourself some prep and wind down time.
```

File: resources/expectations.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Workshop follow-up (1h)
```
Comment (hergaiety) [inline resources/expectations.md:22]: What does this entail?

File: resources/expectations.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Mentors
+
+1. Prepate issues for the workshop (3h) [BONUS]
```
Comment (hergaiety) [inline resources/expectations.md]:
```suggestion
1. Prepare issues for the workshop (3h) [BONUS]
```

**Review by hergaiety — CHANGES_REQUESTED** [review]
This is great! Made a few comments but this looks pretty solid.

File: resources/expectations.md (hunk 3 above)
Comment (locks) [reply resources/expectations.md:22]: Added something!

**Comment by locks** [comment]
This document isn't super exhaustive at the moment. Do we want to merge and iterate, or iterate in the PR and then merge?

**Review by hergaiety — APPROVED** [review]
@locks looks great and should be good to merge :D
(Which I don't have the power to do, hehe)

**Pull request merged**